scalalib/src/mill/scalalib/PublishModule.scala

```diff
@@ -404,6 +404,15 @@ trait PublishModule extends JavaModule { outer =>
             pom() -> s"$baseName.pom"
           )
         }
+      case PackagingType.Aar => Task.Anon {
+          val baseName = baseNameTask()
+          Seq(
+            jar() -> s"$baseName.aar",
+            sourceJar() -> s"$baseName-sources.jar",
+            docJar() -> s"$baseName-javadoc.jar",
+            pom() -> s"$baseName.pom"
+          )
+        }
```

Comment:
This look correct, though there is a slight chance someone typo-ed the `aar` type but the `jar` task still provides a normal JAR file. Maybe, we should also match on the module type and directly use the `androidAar` task?